### DISCUSSION TITLE
“Orders and Returns” page "Reorder" & "Print Order" are not working

### DISCUSSION THREAD

**Discussion by pabodah** [breezefront/community#165] [5. General]
Hello,
Please check the following issue. Thank you.

**Issue**
1. “Orders and Returns” page "Reorder" & "Print Order" links are not working

**Recreate the issue:**
1. Create new CMS page via backend → CONTENT → Elements → Pages
Page title: “Track order”
Content > Edit with Page Builder > Add a “Row” > Add “HTML Code” > “Insert Widget” → Widget Type  = “Orders and Returns”
URL Key: “track-order”
Page in Websites: “All Store Views”
Design > Layout” 2 columns with left bar
meta title = Track order Official Online Store

2. Go to frontend /track-order page
3. Select "Email" for "Find Order By", enter email, order ID, Lastname
4. Click on "Search
5.  Click on "Reorder" link but product is not added to the cart
6. Click on "Print Order" link but the print is not working
[Screencast 2023-06-24 03-22-11.webm](https://github.com/breezefront/community/assets/1543867/716a6120-0707-4d52-802c-1ff8d1ad6df2)

**Comment by vovayatsyuk** [comment]
Thanks for reporting! You can use a built-in page without creating new CMS page and widget. It's available at yoursite.com/sales/guest/form url.

1. Reorder works fine for me, can't reproduce.
2. Fixed in recent commits: 
    - Print dialog: https://github.com/breezefront/module-breeze/commit/1c963abb368dee773dc5b859f233c77f2b336a4b, 
    - Styles improvements: https://github.com/breezefront/theme-frontend-breeze-blank/commit/3ac8066b63e5776a0db1691b55f5f8331e7e4ed3, https://github.com/breezefront/theme-frontend-breeze-evolution/commit/61d1a20721fd39b0a9730f1599eb8994962bc7fc

**Reply by pabodah** [reply]
Thank you for this.
The solution works.
